processes/bootstrap/pmlogger.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package vpp-agent implements the main entry point into the VPP Agent
+// and it is used to build the VPP Agent executable.
+
+package bootstrap
+
+import (
+	"github.com/pkg/errors"
+	"io"
+	"log"
+	"os"
+	"sync"
+
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/logrus"
+)
+
+const defaultLogfilePath = "/var/log/vpp-agent-init.log"
```

Comment:
Not a good default log name. There should be only default log path and name should be derived from process name.